requirements.txt

```diff
@@ -12,6 +12,7 @@ broadbean>=0.9.1
 wrapt
 pandas
 tqdm
-dataclasses  # this won't be needed when we move to python 3.7
+dataclasses;python_version<'3.7'  # this won't be needed when we move to python 3.7
+websockets==8.0.2
```

Comment:
does this need to be reflected in [`environment.yaml`](https://github.com/QCoDeS/Qcodes/blob/99f543dcd671b372a563bd6e8df938e18c3f02fb/environment.yml#L36) ?

Reply:
Yes I think we should pin it there too